src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -42,7 +42,7 @@ type Props = DefaultProps & {
   /**
    * Additional style rules for the tooltip content.
    */
-  popperStyle?: React.CSSProperties;
+  popperStyle?: React.CSSProperties | SerializedStyles;
```

Comment:
are you sure about this? because SerializedStyles returns the cssProperties as a string inside of the property styles?